include/structured/Dialect/Indexing/IR/IndexingOps.td

```diff
@@ -84,7 +92,8 @@ def Indexing_ARangeOp : Indexing_Op<"arange", [
                        Optional<Index>:$step,
                        OptionalAttr<IndexAttr>:$startAttr,
                        OptionalAttr<IndexAttr>:$stopAttr,
-                       OptionalAttr<IndexAttr>:$stepAttr);
+                       OptionalAttr<IndexAttr>:$stepAttr,
+                       DefaultValuedAttr<BoolAttr, "false">:$foldAttr);
```

Comment:
the python wrapper carries the doc for this - all of the docs here are horribly lacking because I've been paying closer attention to the python. sometime next week I will fill out complete `summary`s and `description`s for everything here.